lib/driver.js

```diff
@@ -325,6 +326,12 @@ class AndroidDriver extends BaseDriver {
     await this.grantPermissions();
     this.apkStrings[this.opts.language] = await helpers.pushStrings(
         this.opts.language, this.adb, this.opts);
+
+    // This must run after installing the apk, otherwise it would make the
+    // install to fail. And before running the app.
```

Comment:
grammar nit: you can say `otherwise it would cause the install to fail`, or `otherwise it would make the install fail`